treeherder/webapp/api/bugzilla.py

```diff
@@ -22,18 +22,19 @@ def create_bug(self, request):
                             status=HTTP_400_BAD_REQUEST)
 
         params = request.data
+        description = "Filed by: {}\n\n{}".format(request.user.username, params.get("comment"))
```

Comment:
If comment isn't provided, this will result in:
```
Filed by foo@bar.com

None
```

Instead perhaps use `params.get("comment", "")`, and maybe format like this:
```python
        description = "Filed by: {}\n\n{}".format(
            request.user.username,
            params.get("comment", "")
        )
```